tests/stdlib/tstrmisc.nim

```diff
@@ -0,0 +1,59 @@
+import strmisc, math
```

Comment:
please move the content of `tests/stdlib/tstrmiscs.nim` into here (tstrmiscs.nim was badly named)